content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
+        "php": ">=7.1",
+        "ext-bz2": "*",
+        "ext-curl": "*",
+    },
+}
+{{< /code/copyable >}}
+
 ## Configure Composer
+The [Composer][bp/composer] and [Composer Install][bp/composer-install]
+buildpacks allow for user-set configuration options for Composer.
+
+#### Deprecated: buildpack.yml configuration
+Note that specifying any configurations through `buildpack.yml` configuration
+will be deprecated in PHP language-family buildpack v1.0.0.
+To migrate from using `buildpack.yml` please set the environment variable
+equivalents instead.
+
+
+### Set the Composer version
+To define a version of Composer to use, set `BP_COMPOSER_VERSION` at build
+time. A supported version must be provided, the supported versions can be found
+on the Paketo Composer [releases page][release/composer]. With the `pack` CLI
+this looks like
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_COMPOSER_VERSION="2.3.1"
+{{< /code/copyable >}}
+
+### Set install options
+To define a command line option for `composer install` to run, set the
+`BP_COMPOSER_INSTALL_OPTIONS` environment variable at build-time to a
+space-separated list of flags.
+
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_COMPOSER_INSTALL_OPTIONS="--no-dev --prefer-install=auto"
+{{< /code/copyable >}}
+
+To define **global** `composer install` configurations, set the
+`BP_COMPOSER_INSTALL_GLOBAL` environment variable at build-time to a
+space-separated list of global installation options.
+
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_COMPOSER_INSTALL_GLOBAL="friendsofphp/php-cs-fixer squizlabs/php_codesniffer=*"
+{{< /code/copyable >}}
 
-The following options are configurable in the app's `buildpack.yml`
+### Set a custom Composer vendor directory
+To define a custom vendoring location for Composer packages, users can set the
+`COMPOSER_VENDOR_DIR` environment variable at build-time. It is a
+[Composer-native environment
+variable](https://getcomposer.org/doc/03-cli.md#composer-vendor-dir) setting.
+It must be set as a relative path under the application source directory root.
 
 {{< code/copyable >}}
-composer:
-  # this allows you to specify a version constaint for composer
-  # any valid semver constaints (e.g. 1.* and 1.10.*) are also acceptable
-  version: 1.10.x
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER_VENDOR_DIR="vendor"
+{{< /code/copyable >}}
 
-  # a list of command line install options for composer
-  # default: ["--no-dev"]
-  install_options: ["--no-dev"]
+### Set the `composer.json` path
+To define a custom `composer.json` path, users can set [Composer-native
+environment variable `COMPOSER`](https://getcomposer.org/doc/03-cli.md#composer) at build-time.
 
-  # default: vendor
-  vendor_directory: vendor
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER="composer"
```

Comment:
Could you swap the example to make it more clear that `COMPOSER` should be set to the path of the `composer.json` file?

e.g.
```suggestion
  --env COMPOSER="other-name-composer.json"
```

Reply:
Yea, totally my bad.